Content.Server/Communications/CommunicationsConsoleSystem.cs

```diff
@@ -254,6 +257,9 @@ private void OnSelectAlertLevelMessage(EntityUid uid, CommunicationsConsoleCompo
                 return;
             }
             _chatSystem.DispatchStationAnnouncement(uid, msg, title, colorOverride: comp.AnnouncementColor);
```

Comment:
Should station announcements be loggin these?

Reply:
Nice catch!